### PR TITLE
Split list for production manager

### PR DESCRIPTION
**Pull request by alexisjanvier**
splitPM

### CODE REVIEW

File: apps/admin/src/App.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ import productionManager from "./request/production-manager";
 const App = () => (
     <Admin authProvider={authProvider} dataProvider={dataProvider}>
         {permissions => [
-            <ResourceWithPermissions
+            permissions['dispatcher-requests'] && permissions['dispatcher-requests'].enabled && <ResourceWithPermissions
```
Comment (floo51) [inline apps/admin/src/App.js:16]: This is supposed to work without the condition

File: apps/api/src/user-account/repository.js [hunk 2]
```diff
@@ -124,21 +130,26 @@ const insertOne = async ({ client, data }) => {
...
-                    );
-
-                    await client('production_management_user')
-                        .transacting(trx)
-                        .insert(linksToCreate);
+                    if (
+                        productionManagementIds &&
+                        productionManagementIds.length
+                    ) {
+                        const linksToCreate = productionManagementIds.reduce(
```
Comment (floo51) [inline apps/api/src/user-account/repository.js]: Isn't a reduce with a push on each iteration actually a map?

**Review by floo51 — CHANGES_REQUESTED** [review]

File: apps/admin/src/App.js (hunk 1 above)
Comment (alexisjanvier) [reply apps/admin/src/App.js:16]: Yop, but it do not ^^

Comment (floo51) [reply apps/admin/src/App.js:16]: Meh !

**Review by floo51 — APPROVED** [review]

**Pull request merged**